mainhandler/imageregistryhandlerhelper.go

```diff
@@ -2,27 +2,76 @@ package mainhandler
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 	"math/rand"
+	"strings"
 	"time"
 
 	"github.com/armosec/armoapi-go/armotypes"
+	"github.com/docker/docker/api/types"
 	"github.com/kubescape/operator/utils"
 
 	"github.com/golang/glog"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func (actionHandler *ActionHandler) updateRegistryScanCronJob() error {
-	jobParams := actionHandler.command.GetCronJobParams()
-	if jobParams == nil {
-		glog.Infof("updateRegistryScanCronJob: failed to get jobParams")
-		return fmt.Errorf("failed to get failed to get jobParams")
+func (actionHandler *ActionHandler) updateSecret(sessionObj *utils.SessionObj, secretName string, authConfig *types.AuthConfig) error {
```

Comment:
Do we need to change helm chart RBAC accordingly to allow us creating/updating/deleteing secrets?
Or we already have those permissions?

Reply:
no, we will need to update it